brands/amenity/vending_machine.json

```diff
@@ -168,5 +168,47 @@
       "name": "Ключ здоровья",
       "vending": "water"
     }
+  },
+  "amenity/vending_machine|コカ・コーラ": {
+    "countryCodes": ["jp"],
+    "match": ["shop/vending_machine|コカコーラ"],
+    "nocount": true,
+    "tags": {
+      "amenity": "vending_machine",
+      "brand": "コカ・コーラ",
+      "brand:en": "Coca-Cola",
+      "brand:ja": "コカ・コーラ",
+      "drink": "cola",
+      "name": "コカ・コーラ",
+      "name:en": "Coca-Cola",
+      "name:ja": "コカ・コーラ",
+      "operator": "Nippon Coca Cola Co., Ltd.",
```

Comment:
Is Coca-Cola Japan really the operator of all the vending machines?

Reply:
Yes. I found from [this PDF](https://www.cocacola.co.jp/content/dam/journey/jp/ja/private/2017/10/factsheet/Coca-Cola-Japan-System-Fact-Sheet_2017Q4_EN.PDF).